startupscript/gcp/resource-mount.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# resource-mount.sh
+#
+# Installs gcsfuse if it is not already installed and mount resources.
+#
+# Note that this script is intended to be sourced from the "post-startup.sh" script
+# and is dependent on some functions and variables already being set:
+#
+# - emit (function)
+# - Workbench CLI is installed
+ 
+if ! which gcsfuse >/dev/null 2>&1; then
+  emit "Installing gcsfuse..."
+
+  # install packages needed to install gcsfuse
+  apt-get install -y \
+    fuse \
+    lsb-core
+
+  # Install based on gcloud docs here https://cloud.google.com/storage/docs/gcsfuse-install.
+  export GCSFUSE_REPO="gcsfuse-$(lsb_release -c -s)" \
```

Comment:
Sorry, I know you were just moving this code, but does this variable actually get exported to a child process or is it just used in the next command? If the latter, then please just make it:

```
  readonly GCSFUSE_REPO="gcsfuse-$(lsb_release -c -s)" \
```

I also think this is unnecessarily strung together as a compound AND-ed set of commands. Since this is being called from a script that already has "set -o errexit" enabled, this should just be:

```
  readonly GCSFUSE_REPO="gcsfuse-$(lsb_release -c -s)"
  echo "deb https://packages.cloud.google.com/apt ${GCSFUSE_REPO} main" > /etc/apt/sources.list.d/gcsfuse.list
  curl "https://packages.cloud.google.com/apt/doc/apt-key.gpg" | apt-key add -

  apt-get update
  apt-get install -y gcsfuse
```

I think that reads easier and is more supportable. It diverges less from the gcsfuse install documentation.

Note that in the example above, I changed the `| tee` to just `>` because this is also running as root already. The `| tee` is a holdover from `| sudo tee` which isn't needed here.